### ISSUE TITLE
New Lichess app no longer available for Amazon Fire Max 11 tablet

### ISSUE THREAD

**Issue by anongithubuser12345** [lichess-org/mobile#875]
### What is the problem?

I was on the beta and noticed no updates for a while. Turns out you've made a change which means the (sideloaded) Play Store now says "this tablet isn't compatible with this app". It was last updated in June. It was working fine before (I've previously reported bugs one of which was fixed).

### Steps to reproduce the bug

Try and install

### What did you expect to happen?

App installed

### What happened instead?

App not installed

### Device Info

Amazon Fire Max 11

### Operating System Version

os v8.3.2.4

**Comment by Raptor007** [comment]
The beta is also incompatible with the Titan Pocket on Android 11.  Maybe it requires Android 12?  (I can't imagine what features wouldn't be available in Android 11 to play chess, though.)

**Comment by anongithubuser12345** [comment]
> The beta is also incompatible with the Titan Pocket on Android 11. Maybe it requires Android 12? (I can't imagine what features wouldn't be available in Android 11 to play chess, though.)

No, it's not that.  The app works fine on the tablet if you download and install the apk manually; it's just not available in the Play Store.   Oddly it IS in the Play Store on an older Amazon tablet, possibly because that's got a smaller screen? (Although on that one (HD8) it's locked in portrait mode.) Who knows.    

APK here: https://github.com/lichess-org/mobile/releases

**Comment by Raptor007** [comment]
> > The beta is also incompatible with the Titan Pocket on Android 11. Maybe it requires Android 12? (I can't imagine what features wouldn't be available in Android 11 to play chess, though.)
> 
> No, it's not that. The app works fine on the tablet if you download and install the apk manually; it's just not available in the Play Store. Oddly it IS in the Play Store on an older Amazon tablet, possibly because that's got a smaller screen? (Although on that one (HD8) it's locked in portrait mode.) Who knows.
> 
> APK here: https://github.com/lichess-org/mobile/releases

Interesting!  The Titan Pocket does have an unusual square screen, so perhaps there's some store restriction based on resolution or aspect ratio.

_Edit:_ The APK works!  Thanks.
